adrs/001-organization-payments.md

```diff
@@ -0,0 +1,15 @@
+# Organization and Payments
+
+## Context
+
+In the current Demeter platform, it is necessary to create an organization to link projects and users.
+
+## Decision
+
+- Organization
+
+The organization will be degraded but the fabric will have support for it in legacy driven just for compatibility.
+
+- Payments
+
+Strip integration will happen just when the user wants to upgrade the plan(tier), so the user will be redirected to a screen to set the payment and the strip will be linked to the project. In fabric, the Strip integration will be into the payment driven where in the future new payment methods can be offered.
```

Comment:
```suggestion
Stripe integration will happen just when the user wants to upgrade the plan(tier), so the user will be redirected to a screen to set the payment and the stripe will be linked to the project. In fabric, the Stripe integration will be into the payment driven where in the future new payment methods can be offered.
```

I think here we should also add that a Stripe consumer ID will be created when a project is created right?